bin/cron/with_pg/dms_tasks

```diff
@@ -55,26 +66,20 @@ def main
   dms_client = Aws::DatabaseMigrationService::Client.new
   dms_client.describe_replication_tasks.replication_tasks.each do |replication_task|
     task_name = replication_task.replication_task_identifier
-    next unless task_name.start_with? 'cron'
-
-    task_arn = replication_task.replication_task_arn
-    # TODO(asher): Gracefully handle the case when the DMS task has never been run.
     task_execution_time = replication_task.replication_task_stats.elapsed_time_millis / 1000
-    task_delay = get_delay(task_execution_time)
+    next unless run_task?(task_name, task_execution_time_sec)
 
     ChatClient.message 'infra-dms',
       "Starting DMS task #{task_name} (sleep duration: #{task_delay})...",
       color: 'green'
 
     dms_client.start_replication_task(
       {
-        replication_task_arn: task_arn,
+        replication_task_arn: replication_task.task_arn,
         start_replication_task_type: REPLICATION_TASK_TYPE
       }
     )
-    sleep(task_delay)
-
-    ChatClient.message 'infra-dms', "Finished DMS task #{task_name}.", color: 'green'
```

Comment:
Because it is misleading, as we haven't polled AWS to actually know the task finished (successfully), much less to know the task started (successfully).